hbase-server/src/test/java/org/apache/hadoop/hbase/snapshot/TestRestoreSnapshotHelper.java

```diff
@@ -107,6 +117,8 @@ public void setup() throws Exception {
     conf = TEST_UTIL.getConfiguration();
     setupConf(conf);
     CommonFSUtils.setRootDir(conf, rootDir);
+    // Turn off balancer so it doesn't cut in and mess up our placements.
+    admin.balancerSwitch(false, true);
```

Comment:
since other places in the class just call `TEST_UTIL.getAdmin` you don't have to store the variable. (Or if you store the variable, might as well use it in the testmethod directly.)
```suggestion
    TEST_UTIL.getAdmin().balancerSwitch(false, true);
```